torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -774,6 +775,15 @@ Tensor* TensorExprKernel::computeValue(const torch::jit::Value* v) {
       });
     } break;
 
+    case aten::to: {
+      auto node = v->node();
+      return computeOneOperand("aten_to", v, [node](const ExprHandle& a) {
```

Comment:
AFAIU, `to` might have more than one input, and we seem to be ignoring them here. Should we specially handle or reject such cases?

Reply:
I think we're already rejecting them. I didn't add support for the aten::to overload which takes two Tensors.

Reply:
I could add support for it as a follow up, I think it's less common

Reply:
I didn't mean `to` with two tensors, I mean `to` with optional arguments, like
```
x = y.to(torch.float32, non_blocking=True, copy=True, memory_format=torch.preserve_format)
```
My question was about these cases (which are missing in tests too).

Reply:
copy doesn't apply, we skip fusion in the case that the value is used, which there is a test for. 

memory_format doesn't apply, it's only applied to output tensors as a last step, just as with all other operations (at least when https://github.com/pytorch/pytorch/pull/48264 lands, which i'm trying to). 

and non_blocking i also don't think really applies, but i'm not really sure why it would ever be used or matter

Reply:
What about `pin_memory` in the `to.dtype_layout`? Either way, please add a comment why it is safe to ignore the other arguments. Might be worth adding a test for these too.

Reply:
You're right about pin_memory, I think that's a cpu option to make it faster to copy to gpu. In this case, i'll skip. And agreed about adding comments/tests